internal/test/smoke/system_endpoints_test.go

```diff
@@ -87,20 +95,60 @@ func (s *SuiteSystemEndpoints) TestHostConfExecute() {
 	t := s.T()
 	s.As(RBACSuperAdmin)
 	s.prepareDomainIpa(t)
-
-	t.Log("Calling SystemHostConfWithResponse")
 	domainType := public.RhelIdm
 	s.As(XRHIDSystem, RBACNoPermis)
-	res, err := s.HostConfWithResponse(
-		s.domain.RhelIdm.Servers[0].SubscriptionManagerId.String(),
-		"client."+s.domain.DomainName,
-		builder_api.NewHostConf().
-			WithDomainName(pointy.String(s.domain.DomainName)).
-			WithDomainType(&domainType).
-			Build())
-	require.NoError(t, err)
-	require.NotNil(t, res)
-	require.Equal(t, http.StatusOK, res.StatusCode)
+
+	t.Run("HostConf success", func(t *testing.T) {
+		// Expect success HostConf event to be sent to Pendo
+		mockPendo, ok := s.PendoClient.(*mock_pendo.Pendo)
```

Comment:
observation: I could be wrong, but maybe the type assertion is not necessary if the mock is declared as `*mock_pendo.Pendo` at `SuiteBase`.

Reply:
If it would be `*mock_pendo.Pendo` then it would not be necessary, but the type is the interface and thus it is currently necessary.